site/e2e/tests/createWorkspace.spec.ts

```diff
@@ -147,3 +149,43 @@ test("create workspace with disable_param search params", async ({ page }) => {
   await expect(page.getByLabel(/First parameter/i)).toBeDisabled();
   await expect(page.getByLabel(/Second parameter/i)).toBeDisabled();
 });
+
+test("create docker workspace", async ({ page }) => {
+  test.skip(
+    true,
+    "creating docker containers is currently leaky. They are not cleaned up when the tests are over.",
+  );
+  requiresTerraform();
+  const template = await createTemplate(page, StarterTemplates.STARTER_DOCKER);
+
+  const _ = await createWorkspace(page, template);
+
+  // The workspace agents must be ready before we try to interact with the workspace.
+  await page.waitForSelector(
+    `//div[@role="status"][@data-testid="agent-status-ready"]`,
+    {
+      state: "visible",
+    },
+  );
+
+  // Wait for the terminal button to be visible, and click it.
+  const terminalButton =
+    "//a[@data-testid='terminal'][normalize-space()='Terminal']";
+  await page.waitForSelector(terminalButton, {
+    state: "visible",
+  });
+
+  // We can't click the terminal button because that opens a new tab.
+  // So grab the href, and manually navigate.
+  const terminalPageURL = await page.getAttribute(terminalButton, "href");
```

Comment:
just curious, why do we _need_ it to be in the same page? we have an `openTerminalWindow` helper specifically to make it easy